### PR TITLE
Card

### PR DESCRIPTION
**Pull request by rdthetha**
Added cards

### CODE REVIEW

File: components/card/card.css [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+.img-contain {
+    width: 100%;
+    position: relative;
+}
+
+.tag {
+    padding: 4px 8px;
+    border: 1px solid var(--primary-faint-grey);
+    border-radius: 50px;
```
Comment (kgajriaAgarwal) [inline components/card/card.css:64]: % can be used here in border radius

File: components/card/card.css [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+    font-size: 1rem;
```
Comment (kgajriaAgarwal) [inline components/card/card.css:65]: use % in border radius inspite of px

**Review by kgajriaAgarwal — CHANGES_REQUESTED** [review]
use % in border radius inspite of px , rest everything is good!

**Review by Naman-Saxena1 — CHANGES_REQUESTED** [review]
Buttons on card don't have click animations present, everything else looks good to me!

**Comment by rdthetha** [comment]
> use % in border radius inspite of px , rest everything is good!

Fixed it!

**Comment by rdthetha** [comment]
> Buttons on card don't have click animations present, everything else looks good to me!

Added some basic animations to the button

File: components/card/card.css [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+    box-shadow: rgb(0 0 0 / 35%) 0rem 0.2rem 1.2rem;
+    border-radius: 1rem;
+    background-color: var(--primary-compliment);
+    transition: transform 0.4s ease-in-out;
+    position: relative;
+    display: flex;
+    flex-wrap: column wrap;
+    justify-content: center;
+    position: relative;
+}
```
Comment (Ankur9669) [inline components/card/card.css:22]: You have used position relative two times see in line 21 and 17.

File: components/card/card.html [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+                style="width: 100%;
+  height: 335px;
+  border: 0;
+  transform: scale(1);
+  overflow: hidden;" sandbox="allow-scripts allow-same-origin">
+            </iframe>
```
Comment (Ankur9669) [inline components/card/card.html:101]: You can remove inline styling from here.

File: components/card/card.html [hunk 5]
```diff
@@ -0,0 +1,198 @@
...
+            <div class="section-heading">Cards</div>
+            <div class="section-subhead">Cards has many use cases like showing product information to the user,
+                Displaying a particular content in a orderly format etc.</div>
+            <p class="section-para">In the below example we have made normal card and also card with dismiss button at
+                the top-right .</p>
+            <div class="component-example flex-row flex-align-center flex-gap2 flex-justify-center first">
+                <div class="card-container">
+                    <div class="img-contain">
+                        <img src="https://picsum.photos/400" alt="Photo">
+                    </div>
```
Comment (Ankur9669) [inline components/card/card.html:55]: img does not have closing /

File: components/card/card.html [hunk 6]
```diff
@@ -0,0 +1,198 @@
...
+                    style="width: 100%;
+  height: 335px;
+  border: 0;
+  transform: scale(1);
+  overflow: hidden;" sandbox="allow-scripts allow-same-origin">
```
Comment (Ankur9669) [inline components/card/card.html:151]: Same inline styling could be removed

**Review by Ankur9669 — CHANGES_REQUESTED** [review]
I would like to suggest few changes...
1. In vertical cards the left and right padding could be increased.
2. In the iframes the indentation needs to be improved.

File: components/card/card.html (hunk 5 above)
Comment (rdthetha) [reply components/card/card.html:55]: in HTML5 the ending tag is not required

File: components/card/card.html (hunk 4 above)
Comment (rdthetha) [reply components/card/card.html:101]: Thank you for the suggestion ! Removed the inline styling

File: components/card/card.css (hunk 3 above)
Comment (rdthetha) [reply components/card/card.css:22]: fixed!

File: components/card/card.html (hunk 6 above)
Comment (rdthetha) [reply components/card/card.html:151]: Removed!

**Pull request merged**